stonk_market_mayhem.py

```
@@ -0,0 +1,76 @@
+
+def show_menu():
+    print("1. View Stocks")
+    print("2. Buy Stock")
+    print("3. Sell Stock")
+    print("4. View Portfolio")
+    print("5. Exit")
+
+def view_stocks():
+    for stock, price in stocks.items():
+        # Prices fluctuate wildly each time you look
+        new_price = price + random.randint(-100, 100)
```

Ruff (0.8.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)